src/index.js

```diff
@@ -0,0 +1,78 @@
+import './style.css';
+
+const pressEnter = document.getElementById('press-enter');
+const hoverText = document.getElementById('button-hover');
+
+const showToDo = document.getElementById('task-list');
+const myTask = document.getElementById('my-task');
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+
+// Hover message when hover over enter icon
+pressEnter.addEventListener('mouseover', () => {
+  hoverText.style.display = 'block';
+});
+
+pressEnter.addEventListener('mouseout', () => {
+  hoverText.style.display = 'none';
+});
+
+// Local Storage
+class ToDo {
+  constructor() {
+    this.taskList = JSON.parse(localStorage.getItem('storage-task')) || [];
+  }
+
+  addTask(description) {
+    const updateTask = [
+      ...this.taskList,
+      { description },
```

Comment:
- [Optional] Great job for implementing previous code reviewer suggestions, however, you have missed one requirement, which is your object has to include an index and completed :+1:  please update to track the completion and identity each to do the task as the below sample
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```